scripts/get.template.sh

```diff
@@ -7,7 +7,16 @@ VERSION="VERSION_PLACEHOLDER"
 echo "Downloading Semaphore CLI release ${VERSION} for ${OS}_${ARCH} ..."
 echo ""
 
-curl --fail -L "https://github.com/semaphoreci/cli/releases/download/${VERSION}/sem_${OS}_${ARCH}.tar.gz" -o /tmp/sem.tar.gz
+readonly TMP_DIR="$(mktemp -p . -d -t sem-XXXX)"
```

Comment:
the flag `-p .` will not work on macOS. I'll look for something else.

Reply:
Removing this works on macOS, but I cannot test it on Ubuntu.

Reply:
Tested on Ubuntu. It works.